### PR TITLE
2021 03 12 adrian01c practice html

### PR DESCRIPTION
**Pull request by Adrian01C**

### CODE REVIEW

**Review by marianzburlea — APPROVED** [review]
💯

**Review by andreimaff — APPROVED** [review]
nice 👍🏼

File: web/adrian01c/html/track/index.html [hunk 1]
```diff
@@ -4,10 +4,18 @@
...
   </head>
-  <body>
-    <h1>This is track</h1>
-    <h2>I want to create a dropdown menu but i have to use css and java for that</h2>
-  </body>
+  <body> 
+    <h1 style="background-color:DodgerBlue;">Did Romania melt the Russian tanks with a laser in 1968?</h1>
+    <h2 style="color: brown;">As the story goes, in November of 1968, Soviet forces were massing on Romania’s eastern border, in preparation for a military intervention. This was a reprisal prompted by Ceausescu’s defiant stance, particularly with regard to the Warsaw Pact invasion of Czechoslovakia earlier that year. That is when, supposedly, a top secret super powerful laser sprung into action, melting/vaporizing several tanks and aircraft, scaring the Soviets shitless and dissuading them from their belligerent intentions.</h2>
+    <img src="https://qph.fs.quoracdn.net/main-qimg-fccdedd9884c944d0b7f3b08ff3f0764.webp" alt="Real Laser Tank" width="600" height="200">
+    <button>See here more discussions on Quora about it</button> <link rel="stylesheet"
```
Comment (RalucaRbc) [inline web/adrian01c/html/track/index.html]: Should have a new line between `</button>` and `<link... >`

File: web/adrian01c/html/track/index.html [hunk 2]
```diff
@@ -4,10 +4,18 @@
...
-    <h2>I want to create a dropdown menu but i have to use css and java for that</h2>
-  </body>
+  <body> 
+    <h1 style="background-color:DodgerBlue;">Did Romania melt the Russian tanks with a laser in 1968?</h1>
+    <h2 style="color: brown;">As the story goes, in November of 1968, Soviet forces were massing on Romania’s eastern border, in preparation for a military intervention. This was a reprisal prompted by Ceausescu’s defiant stance, particularly with regard to the Warsaw Pact invasion of Czechoslovakia earlier that year. That is when, supposedly, a top secret super powerful laser sprung into action, melting/vaporizing several tanks and aircraft, scaring the Soviets shitless and dissuading them from their belligerent intentions.</h2>
+    <img src="https://qph.fs.quoracdn.net/main-qimg-fccdedd9884c944d0b7f3b08ff3f0764.webp" alt="Real Laser Tank" width="600" height="200">
+    <button>See here more discussions on Quora about it</button> <link rel="stylesheet" 
+    
+      href="https://www.quora.com/Did-Romania-melt-Russian-tanks-with-a-laser-in-1968">
+    Did-Romania-melt-Russian-tanks-with-a-laser-in-1968">
```
Comment (RalucaRbc) [inline web/adrian01c/html/track/index.html]: You should delete this code line as well 😄

**Review by RalucaRbc — APPROVED** [review]

**Review by psyhedeliq — APPROVED** [review]
👏

File: web/adrian01c/html/track/index.html (hunk 1 above)
Comment (QuintyHH) [inline web/adrian01c/html/track/index.html]: No.... no no no, that's not what `<link>` does, google `<a href>` instead.

File: web/adrian01c/html/track/index.html (hunk 2 above)
Comment (QuintyHH) [inline web/adrian01c/html/track/index.html]: replace everything between lines 13 and 16 with 
```
    <a href="https://www.quora.com/Did-Romania-melt-Russian-tanks-with-a-laser-in-1968">
        <button>Did Romania melt Russian Tanks with lasers in 1968?</button>
    </a>
```

**Review by QuintyHH — CHANGES_REQUESTED** [review]

File: web/adrian01c/html/track/index.html (hunk 1 above)
Comment (marianzburlea) [reply web/adrian01c/html/track/index.html]: indeed

**Review by marianzburlea — CHANGES_REQUESTED** [review]
:) What @RalucaRbc said

**Review by psyhedeliq — APPROVED** [review]

**Review by marianzburlea — APPROVED** [review]

**Pull request merged**